extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/datamasking/DataMaskingTestBase.scala

```diff
@@ -260,4 +260,50 @@ trait DataMaskingTestBase extends AnyFunSuite with SparkSessionProvider with Bef
         Seq(Row(md5Hex("1"), "hello")))
     }
   }
+
+  test("test MASK rule with non-English character set") {
```

Comment:
hi @huangzhir, these tests are not verifying any code we changed. please use a normal query like `select col0 from t1`, where `t1.col0` has a masker configuration with show_xxx_4. It shall be done by the masking rule automatically instead of what we do here manually

Reply:
you can refer to other tests in this test base